.github/workflows/check-shell.yml

```
@@ -0,0 +1,29 @@
+name: Check shell.nix
```

```suggestion
name: "Check shell"
```

To make it consistent with the others